packages/stark-ui/src/modules/dropdown/components/dropdown.component.spec.ts

```diff
@@ -0,0 +1,286 @@
+//tslint:disable:no-big-function no-commented-code completed-docs
+import { Component, DebugElement, NO_ERRORS_SCHEMA, ViewChild } from "@angular/core";
+import { ComponentFixture, TestBed, async } from "@angular/core/testing";
+import { MatOptionModule, MatSelectModule } from "@angular/material";
+import { STARK_LOGGING_SERVICE } from "@nationalbankbelgium/stark-core";
+import { StarkDropdownComponent } from "./dropdown.component";
+import { CommonModule } from "@angular/common";
+import { By } from "@angular/platform-browser";
+import { MockStarkLoggingService } from "@nationalbankbelgium/stark-core/testing";
+import { TranslateModule, TranslateService } from "@ngx-translate/core";
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+
+@Component({
+	selector: `host-component`,
+	template: `
+		<stark-dropdown [options]="options" [label]="label" [placeholder]="placeholder"
+						[value]="value" [selectionChange]="dropdownSelectionChanged"
+						[dropdownId]="dropdownId"></stark-dropdown>`
+})
+class TestHostComponent {
+	@ViewChild(StarkDropdownComponent)
+	public dropdownComponent: StarkDropdownComponent;
+	public dropdownId: string;
+	public header?: string;
+	public label?: string;
+	public options: any[];
+	public placeholder: string;
+	public value: any | any[];
+	public dropdownSelectionChanged: Function;
+	public required?: boolean;
+}
+
+fdescribe("DropdownComponent", () => {
+	let component: StarkDropdownComponent;
+	let hostComponent: TestHostComponent;
+	let hostFixture: ComponentFixture<TestHostComponent>;
+
+	const simpleOptions: string[] = ["1", "2", "3"];
+	const dropdownLabel: string = "dropdown label";
+	const dropdownPlaceholder: string = "dropdown placeholder";
+	const dropdownValue: string = "dropdownValue";
+	const dropdownId: string = "dropdownId";
+	const dropdownOnChange: any = () => {
+		return "dummyDropdownOnChange";
+	};
+
+	beforeEach(async(() => {
+		return TestBed.configureTestingModule({
+			imports: [CommonModule, MatSelectModule, MatOptionModule, TranslateModule.forRoot(),BrowserAnimationsModule],
+			declarations: [StarkDropdownComponent, TestHostComponent],
+			providers: [{ provide: STARK_LOGGING_SERVICE, useValue: new MockStarkLoggingService() }, TranslateService],
+			schemas: [NO_ERRORS_SCHEMA] // tells the Angular compiler to ignore unrecognized elements and attributes (svgIcon)
```

Comment:
Is this needed? I don't think so since there are no MatIcons being used in the template